CHANGELOG.md

```
@@ -1,3 +1,10 @@
+# [0.12.0-beta.1](https://github.com/counterfake/counterfake-dashboard/compare/v0.11.0...v0.12.0-beta.1) (2025-10-08)
+
+
+### Features
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)